### ISSUE TITLE
weight update for step=0,epoch=0 get's missed.

### ISSUE THREAD

**Issue by gurdeepmaurya** [facebookresearch/barlowtwins#45]
The function adjust_learning_rate() sets learning rate to 0 for step=0, epoch=0, Thereby missing out on the very first weight update.

**Comment by jingli9111** [comment]
@gurdeepmaurya 
Nice catch! I think this is common for any method using warm-up. The first few iterations all use very small learning rate.
But it still does something useful: The batchnorm layers can still collect statistics during the forward pass.

**Comment by gurdeepmaurya** [comment]
Yeah. Closing this issue since its fix is not that big of an issue.
